pom.xml

```diff
@@ -7,7 +7,7 @@
 
     <groupId>de.flux.playground</groupId>
     <artifactId>deckcompare</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>

```

Comment:
Why did you change the version? The project right now is not even in a real functional stage... XD